CONTRIBUTING.md

```diff
@@ -242,6 +242,8 @@ In any case, thank you very much for your contributions!
 }
 ```
 
+If your extractor supports cookies, use the `cookiefile` or `cookiesfrombrowser` keys in the same way as the `--cookies` or `--cookies-from-browser` options.
```

Comment:
Why not (L237)
```diff
-**Tip:** To test extractors that require login information, create a file `test/local_parameters.json` and add `"usenetrc": true` or your username and password in it:
+**Tip:** To test extractors that require login information, create a file `test/local_parameters.json` and add `"usenetrc": true` or your `username`&`password` or `cookiefile`/`cookiesfrombrowser` in it:

Reply:
Well, I have some points about this:

- Compared with username/password, cookies are not plaintext credential.
- Specifying netrc is another way to config username/password.
- The name of the "cookiefile" key is not from the "--cookies" option with removing dashes. It's a bit more special than others.

Your opinion?

Reply:
Hi, @pukkandan ! I've seen yt-dlp/yt-dlp@42a7a7ceac8de0fcf1605e3d4e45a24f47101837 . Now I agree with you. We have only one config file, so splitting options may confuse new developers. Your suggestion makes things better.